src/AppMenuButtonGroup.cc

```
@@ -1066,7 +1071,7 @@ void AppMenuButtonGroup::onSubMenuReady(QMenu *menu)
         }
     }
 
-    if (m_buttonIndexWaitingForPopup == -1 || !m_appMenuModel || !m_appMenuModel->menu()) {
+    if (m_buttonIndexWaitingForPopup < 0 || !m_appMenuModel || !m_appMenuModel->menu()) {
```

Clang (14.0.6)

[warning] implicit conversion 'Material::AppMenuModel *' -> bool

(readability-implicit-bool-conversion)